projects/igniteui-angular/src/lib/core/styles/components/navdrawer/_navdrawer-theme.scss

```diff
@@ -251,17 +255,18 @@
     %item {
         position: relative;
         display: flex;
+        align-items: center;
         flex-flow: row nowrap;
         color: --var($theme, 'item-text-color');
         max-height: rem(48px);
-        padding: rem(12px) rem(16px);
+        min-width: rem(32px);
+        padding: rem(12px) rem(8px);
         cursor: pointer;
-        align-items: center;
         user-select: none;
         outline: transparent;
         white-space: nowrap;
         border-radius: --var($theme, 'item-border-radius');
-        margin: 8px;
+        margin: rem(8px) rem(4px);
```

Comment:
BTW, I'm wondering why you changed the margin. I feel like it's narrow. The margin of left/right became 12px. It's smaller than 16px, which is defined by [Material Design guidelines](https://material.io/design/components/navigation-drawer.html#specs)
![drawer-margin](https://user-images.githubusercontent.com/15909409/62987690-0758e800-be7c-11e9-82bf-f325ae614e75.png)